discord/bot.py

```diff
@@ -267,9 +264,9 @@ def _check_command(cmd: ApplicationCommand, match: Mapping[str, Any]) -> bool:
                             for data in match["options"]
                             if data["name"] == subcommand.name
                         ),
-                        MISSING,
+                        None,
                     )
-                    if match_ is not MISSING and _check_command(subcommand, match_):
+                    if match_ and _check_command(subcommand, match_):
```

Comment:
I don't think this has anything to do with the cog attribute being MISSING, the MISSING value here is used for matching commands.